tests/follow.test.js

```diff
@@ -41,28 +40,31 @@ describe('TEST USER FOLLOW', () => {
       });
   });
 
-  it('It should return a 201 if a user is been followed successfully', done => {
+  it('It should return a 201 if a user followed successfully', done => {
```

Comment:
Can you refactor this test description to clearly describe what this is testing?